core/camel-base/src/main/java/org/apache/camel/impl/health/RouteHealthCheck.java

```diff
@@ -84,21 +84,23 @@ protected void doCall(HealthCheckResultBuilder builder, Map<String, Object> opti
                 builder.up();
             }
 
-            if (builder.state() != State.DOWN) {
-                // If JMX is enabled, use the Managed MBeans to determine route
-                // health based on performance counters.
-                ManagedRouteMBean managedRoute = context.getExtension(ManagedCamelContext.class).getManagedRoute(route.getId());
+			if (builder.state() != State.DOWN) {
```

Comment:
I think the indentation is wrong. Please run this with the profile sourcecheck enabled. You should see warnings.

Reply:
OK, I'll update it

Reply:
I've run this from camel/core/camel-base but it's not reporting any errors. Is there something else I should check?
`mvn clean install -Psourcecheck`

Reply:
Oh, I wasn't looking for warnings. I'll fix them now. Sorry for my confusion.